data/mods/Xedra_Evolved/eocs/magic_potential_eoc.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_MAGIC_FROM_WOUNDS",
+    "recurrence": [ "8 h", "16 h" ],
+    "//": "if you got badly wounded, you may obtain some powers; threshold right now is 50% of your hp, but may be decreased in the future",
+    "condition": { "math": [ "u_hp() / 6", "==", "u_hp_max('torso') / 2" ] },
+    "effect": [ { "math": [ "magic_potential", "++" ] } ]
+  },
+  {
```

Comment:
What about also setting one from morale dropping significantly?  Either a -100 effect or just getting to the depression effect's like too sad to read?

Reply:
Yeah, this part require a lot of work; i plan to cheat a bit and add "witnessed portal storm" conduct, so there would be at least some guarantee to player to not left for a long time without a class
Also i may need to make sequence start not from magic_potential > 0, but something like > 2, but it is a separate issue i need to test